web-server/v0.4/src/routes/Dashboard/RunComparison.js

```diff
@@ -391,9 +527,48 @@ export default class RunComparison extends ReactJS.Component {
 
     const action = (
       <div>
-        <Button type="primary" onClick={window.print}>
-          {'Share'}
+        <Button type="primary" onClick={this.showModal}>
+          {'Export'}
         </Button>
+        <Modal
+          title="Export to PDF"
+          visible={this.state.modalVisible}
+          onOk={this.handleOk}
+          okText={'Save'}
+          onCancel={this.handleCancel}
+        >
+          <Spin spinning={this.state.generatingPdf}>
+            <Form>
+              <Form.Item
+                colon={false}
+                label="File Name"
+                extra="(Optional) Selected results will be used if left blank."
```

Comment:
We render a timestamp instead of the selected results here, can we update this statement to reflect that?